src/agent/service/com.wazuh.agent.plist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+ <plist version="1.0">
+     <dict>
+         <key>Label</key>
+         <string>com.wazuh.agent</string>
+
+         <key>ProgramArguments</key>
+         <array>
+             <string>/path/to/executable/wazuh-agent</string>
+             <string>--start</string>
+         </array>
+
+        <key>WorkingDirectory</key>
+        <string>/path/to/working/directory</string>
+
+        <key>StandardOutPath</key>
+        <string>/var/log/wazuh-output.log</string>
+
+        <key>StandardErrorPath</key>
+        <string>/var/log/wazuh-error.log</string>
+
+         <key>RunAtLoad</key>
+         <true/>
+
+         <key>KeepAlive</key>
+         <true/>
+     </dict>
+ </plist>'
```

Comment:
Is the trailing `'` necessary?

There's a missing line at the end.

Reply:
No, that's a typo